data/en/banner.yml

```diff
@@ -3,9 +3,9 @@ banner:
   enable : true
   bg_image_webp : "images/backgrounds/NASA_GulfofMex.webp"
   bg_image : "images/backgrounds/NASA_GulfofMex.jpg"
-  image: "images/assets/logos/logo.png"
+  image: "images/assets/logos/Rotationaltxt.png"
```

Comment:
Out of curiosity, have you double-checked that this will look ok in other resolutions? E.g. mobile/tablet?